### ISSUE TITLE
Count story points in milestones

### ISSUE THREAD

**Issue by koppor** [banyan/github-story-points#11]
In a milestone, the story points should also be summarized.

![image](https://user-images.githubusercontent.com/1366654/69853590-96e9a180-1287-11ea-98dd-63745ff4c518.png)

**Comment by scasplte2** [comment]
+1 to this request
Really enjoy the extension and this would feature would be super convenient

**Comment by atreeon** [comment]
a great request (I only use projects now to add up the points using this feature...always use milestones)

**Comment by ericboucher** [comment]
This one is trickier since we cannot simply rely on info from the page. We would likely need to send an API request to get all the issues in the milestone and their labels/titles and then count story points.

**Comment by koppor** [comment]
> This one is trickier since we cannot simply rely on info from the page. We would likely need to send an API request to get all the issues in the milestone and their labels/titles and then count story points.

Naively I would have said, that the points go after the open or closed label - depending on what the user selected - and "just" summarizing the content. No total sum displayed (for a first version)

Am I missing some hidden tickets then?

**Comment by ericboucher** [comment]
Indeed, as a first version, we could just summarize what's available on the page. And show the number of story points on the page you have open.

On the milestone view, we either see the closed or the open issues. So to do more we would need a request, unfortunately.
